--- ./include/ext2fs/ext2fs.h.orig	2006-06-01 19:24:53.000000000 +0200
+++ ./include/ext2fs/ext2fs.h	2007-07-23 01:23:19.000000000 +0200
@@ -50,17 +50,8 @@
 #if EXT2_FLAT_INCLUDES
 #include "e2_types.h"
 #else
-#include <asm/types.h>
-#if !defined(__GNUC__) || defined(__STRICT_ANSI__)  /* asm/types.h already defines __s64 and __u64 otherwise */
-#if SIZEOF_LONG == 8
-typedef __signed__ long __s64;
-typedef unsigned long __u64;
-#elif SIZEOF_LONG_LONG == 8 || \
-  defined(__GNUC__) && (((~0UL) == 0xffffffff) || defined(__i386__))
-typedef __signed__ long long __s64;
-typedef unsigned long long __u64;
-#endif /* SIZEOF_LONG == 8 */
-#endif
+#include <ext2fs/ext2_types.h>
+#include <ext2fs/ext2_fs.h>
 #endif /* EXT2_FLAT_INCLUDES */
 
 typedef __u32		blk_t;
