--- ./man/Makefile.in.orig	2010-01-02 18:40:05.000000000 +0100
+++ ./man/Makefile.in	2010-01-02 18:41:00.000000000 +0100
@@ -218,7 +218,7 @@
 	faillog.5 faillog.8 getspnam.3 gpasswd.1 groupadd.8 groupdel.8 \
 	groupmems.8 groupmod.8 groups.1 grpck.8 grpconv.8 grpunconv.8 \
 	gshadow.5 lastlog.8 login.1 login.defs.5 logoutd.8 newgrp.1 \
-	newusers.8 nologin.8 passwd.1 passwd.5 pwck.8 pwconv.8 \
+	newusers.8 nologin.8 passwd.1 pwck.8 pwconv.8 \
 	pwunconv.8 sg.1 shadow.3 shadow.5 su.1 suauth.5 useradd.8 \
 	userdel.8 usermod.8 vigr.8 vipw.8 $(am__append_1)
 man_nopam = \
@@ -253,7 +253,6 @@
 	newusers.8.xml \
 	nologin.8.xml \
 	passwd.1.xml \
-	passwd.5.xml \
 	porttime.5.xml \
 	pwck.8.xml \
 	pwconv.8.xml \
