diff -dur libtunepimp-0.3.0/lib/threads/posix/thread.cpp libtunepimp-0.3.0/lib/threads/posix/thread.cpp
--- libtunepimp-0.3.0/lib/threads/posix/thread.cpp	2004-01-11 10:27:52.000000000 +0100
+++ libtunepimp-0.3.0/lib/threads/posix/thread.cpp	2006-01-04 23:54:02.000000000 +0100
@@ -24,6 +24,8 @@
    $Id: thread.cpp,v 1.6 2003/10/05 02:21:52 robert Exp $
 
 ----------------------------------------------------------------------------*/
+#include <pthread.h>
+#include <signal.h>
 #include <stdio.h>
 #include <errno.h>
 #include <string.h>
