--- ./src/hd/kbd.c.orig	2007-06-01 00:12:35.000000000 +0200
+++ ./src/hd/kbd.c	2007-06-01 00:13:02.000000000 +0200
@@ -142,11 +142,13 @@
   }
 
   if(!dev && (fd = open(DEV_CONSOLE, O_RDWR | O_NONBLOCK | O_NOCTTY)) >= 0) {
+#if 0
     if(ioctl(fd, TIOCGDEV, &u) != -1) {
       tty_major = (u >> 8) & 0xfff;
       tty_minor = (u & 0xff) | ((u >> 12) & 0xfff00);
       ADD2LOG(DEV_CONSOLE ": major %u, minor %u\n", tty_major, tty_minor);
     }
+#endif
 
     if(tty_major == 229 /* iseries hvc */) {
       if (tty_minor >= 128) {
