--- grml-x.zsh	2007-06-03 18:16:20.000000000 +0200
+++ grml-x.zsh	2007-06-03 18:43:05.000000000 +0200
@@ -22,13 +22,8 @@
   fi
 
   # some functions like getBootParam
-  if [ -r /etc/grml/script-functions -a -r /etc/grml/sh-lib ] ; then
-     source /etc/grml/script-functions
-     source /etc/grml/sh-lib
-  else
-    echo 'Error: sourcing function files failed. Exiting.'
-    exit 1
-  fi
+INC:functions
+INC:shlib
 
   check4root &>/dev/null && ROOT='1' || ROOT=''
 # }}}
