Benjamin Schieder:
	hotfix to compile vpswitch
--- compiz-fusion-plugins-main-0.6.0/src/vpswitch/vpswitch.c	2007-10-15 19:43:43.000000000 +0200
+++ src.compiz-fusion-plugins-main.1202658584.28593.1429719530/compiz-fusion-plugins-main-0.6.0/src/vpswitch/vpswitch.c	2008-02-10 16:58:35.000000000 +0100
@@ -28,6 +28,8 @@
 
 #include <compiz.h>
 #include <string.h>
+#define XK_LATIN1
+#define XK_MISCELLANY
 #include <X11/keysymdef.h>
 #include "vpswitch_options.h"
 
