--- ./c++/etip.h.in.orig	2008-06-07 10:44:43.000000000 +0200
+++ ./c++/etip.h.in	2008-06-07 10:46:43.000000000 +0200
@@ -335,7 +335,7 @@
 };
 
 #if !((defined(__GNUG__) && defined(__EXCEPTIONS)) || defined(__SUNPRO_CC))
-#  include <iostream.h>
+#  include <iostream>
    extern "C" void exit(int);
 #endif
 
@@ -354,7 +354,7 @@
 #endif
 #else
   if (e)
-    cerr << e->message << endl;
+    std::cerr << e->message << std::endl;
   exit(0);
 #endif
 }
