Match the definition in malloc.c

diff -dur uClibc-0.9.30-rc2/libc/stdlib/malloc/malloc.h src.uclibc.1224243683.3870.4030698952/uClibc-0.9.30-rc2/libc/stdlib/malloc/malloc.h
--- uClibc-0.9.30-rc2/libc/stdlib/malloc/malloc.h	2008-10-11 10:52:58.000000000 +0200
+++ src.uclibc.1224243683.3870.4030698952/uClibc-0.9.30-rc2/libc/stdlib/malloc/malloc.h	2008-10-17 14:24:36.000000000 +0200
@@ -77,7 +77,7 @@
 /* A heap used for allocating malloc_mmb structures.  We could allocate
    them from the main heap, but that tends to cause heap fragmentation in
    annoying ways.  */
-extern struct heap __malloc_mmb_heap;
+extern struct heap_free_area *__malloc_mmb_heap;
 
 /* Define MALLOC_MMB_DEBUGGING to cause malloc to emit debugging info about
    about mmap block allocation/freeing by the `uclinux broken munmap' code
