--- ./Makefile.am.orig	2009-03-25 09:11:12.000000000 +0100
+++ ./Makefile.am	2009-03-25 09:11:24.000000000 +0100
@@ -1,8 +1,7 @@
 AUTOMAKE_OPTIONS = 1.3 foreign
 
 SUBDIRS = hdsploader mixartloader pcxhrloader usx2yloader vxloader echoaudio \
-	  asihpi emi_26_62 emu korg1212 maestro3 multisound sb16 wavefront \
-	  ymfpci aica
+	  asihpi emi_26_62 emu multisound sb16 wavefront aica
 
 EXTRA_DIST = depcomp
 
--- ./Makefile.in.orig	2009-03-25 09:11:27.000000000 +0100
+++ ./Makefile.in	2009-03-25 09:11:38.000000000 +0100
@@ -158,8 +158,7 @@
 target_alias = @target_alias@
 AUTOMAKE_OPTIONS = 1.3 foreign
 SUBDIRS = hdsploader mixartloader pcxhrloader usx2yloader vxloader echoaudio \
-	  asihpi emi_26_62 emu korg1212 maestro3 multisound sb16 wavefront \
-	  ymfpci aica
+	  asihpi emi_26_62 emu multisound sb16 wavefront aica
 
 EXTRA_DIST = depcomp
 all: all-recursive
