--- strace-4.5.19/configure.orig	2010-01-16 22:43:05.000000000 +0100
+++ strace-4.5.19/configure	2010-01-16 22:43:14.000000000 +0100
@@ -5880,7 +5880,7 @@
 do :
   as_ac_Header=`$as_echo "ac_cv_header_$ac_header" | $as_tr_sh`
 ac_fn_c_check_header_compile "$LINENO" "$ac_header" "$as_ac_Header" "#include <stddef.h>
-#include <linux/socket.h>
+#include <sys/socket.h>
 "
 eval as_val=\$$as_ac_Header
    if test "x$as_val" = x""yes; then :
